src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -585,6 +592,15 @@ public Builder appCommandOutputFormat(String appCommandOutputFormat) {
     }
 
     /**
+     * Sets structured json logs for the stderr output. Supported values include 'never' (default),
+     * 'always', 'terminal', etc.
+     */
+    public Builder appCommandShowStructuredLogs(String appCommandShowStructuredLogs) {
```

Comment:
Given that, I agree that "show" beats "use". However, I still think "appCommand" is unnecessary.